src/Authentication/LoginHandler.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Bugcache\Authentication;
+
+use Aerys\Bootable;
+use Aerys\Logger;
+use Aerys\ParsedBody;
+use Aerys\Request;
+use Aerys\Response;
+use Aerys\Server;
+use Aerys\ServerObserver;
+use Aerys\Session;
+use Amp\Promise;
+use Amp\Success;
+use Bugcache\Mustache;
+use Bugcache\SessionKeys;
+use Bugcache\Storage\AuthenticationRepository;
+use Bugcache\Storage\UserRepository;
+use function Aerys\parseBody;
+use function Amp\resolve;
+
```

Comment:
I won't do this until PhpStorm has full support (might have with PhpStorm 11?). I don't want an editor full of red errors.

It's also manual work that's required instead of using autoimporting.

How often do you actually read import statements?!